cluster/images/hyperkube/master-multi.json

```diff
@@ -21,12 +29,19 @@
       "command": [
               "/hyperkube",
               "apiserver",
+              "--admission-control=NamespaceLifecycle,SecurityContextDeny,LimitRanger,ServiceAccount,ResourceQuota",
```

Comment:
Ah, I see that you looked at local-up-cluster.  I'm going to make a PR to remove the logic around this admission controller in that script and just allow security context by default.